packages/react-components/react-combobox/src/components/Combobox/Combobox.test.tsx

```diff
@@ -68,6 +68,74 @@ describe('Combobox', () => {
     expect(container.querySelector('[role=listbox]')).not.toBeNull();
   });
 
+  /* Expand icon name */
+  it('Sets the chevron button name off the aria-label prop', () => {
+    const { container } = render(
+      <Combobox aria-label="test label">
+        <Option>Red</Option>
+        <Option>Green</Option>
+        <Option>Blue</Option>
+      </Combobox>,
+    );
+
+    const chevronButton = container.querySelector('[role=button]');
+    expect(chevronButton?.getAttribute('aria-label')).toEqual('Open test label');
+  });
+
+  it('Sets the chevron button name off the aria-labelledby prop', () => {
+    const { container } = render(
+      <Combobox aria-labelledby="testId">
+        <Option>Red</Option>
+        <Option>Green</Option>
+        <Option>Blue</Option>
+      </Combobox>,
+    );
+
+    const chevronButton = container.querySelector('[role=button]');
+    const chevronId = chevronButton?.id;
+    expect(chevronButton?.getAttribute('aria-label')).toEqual('Open');
+    expect(chevronButton?.getAttribute('aria-labelledby')).toEqual(chevronId + ' testId');
+  });
+
+  it('Defaults to "Open" as the chevron label', () => {
+    const { container } = render(
+      <Combobox>
+        <Option>Red</Option>
+        <Option>Green</Option>
+        <Option>Blue</Option>
+      </Combobox>,
+    );
+
+    const chevronButton = container.querySelector('[role=button]');
+    expect(chevronButton?.getAttribute('aria-label')).toEqual('Open');
+    expect(chevronButton?.getAttribute('aria-labelledby')).toBeFalsy();
+  });
+
+  it('Respects author-provided labels for the chevron button', () => {
+    const renderedCombobox = render(
+      <Combobox expandIcon={{ 'aria-label': 'test label' }}>
+        <Option>Red</Option>
+        <Option>Green</Option>
+        <Option>Blue</Option>
+      </Combobox>,
+    );
+
+    const chevronButton = renderedCombobox.container.querySelector('[role=button]');
+    expect(chevronButton?.getAttribute('aria-label')).toEqual('test label');
+    expect(chevronButton?.getAttribute('aria-labelledby')).toBeFalsy();
+
+    renderedCombobox.rerender(
+      <Combobox expandIcon={{ 'aria-labelledby': 'testId' }}>
+        <Option>Red</Option>
+        <Option>Green</Option>
+        <Option>Blue</Option>
+      </Combobox>,
+    );
+
+    expect(chevronButton?.getAttribute('aria-label')).toBeFalsy();
+    expect(chevronButton?.getAttribute('aria-labelledby')).toEqual('testId');
+  });
+
```

Comment:
Can you add a test where you're setting both the top-level and the `expandIcon` specific `aria-label` and `aria-labelledby` to ensure the specific one wins over the top-level one?

Reply:
I'll bake it into this test -- seems relevant, since this is the test to make sure author-provided chevron props override others.